subsys/pm/policy.c

```diff
@@ -213,6 +226,22 @@ static void update_next_event(uint32_t cyc)
 	next_event_cyc = new_next_event_cyc;
 }
 
+#ifdef CONFIG_PM_LOW_LATENCY_EVENTS
+int32_t pm_policy_next_low_latency_event_ticks(void)
+{
+	int32_t cyc_evt = -1;
+
+	if ((next_low_latency_event) && (next_low_latency_event->value_cyc > 0)) {
+		cyc_evt = next_low_latency_event->value_cyc - k_cycle_get_32();
```

Comment:
To meet the API requirements, this statement should not result in a negative number:
```
cyc_evt = next_low_latency_event->value_cyc - k_cycle_get_32();
cyc_evt = MAX(0, cyc_evt);
```